.env.production.sample

```diff
@@ -44,6 +44,16 @@ SMTP_FROM_ADDRESS=notifications@example.com
 # S3_PROTOCOL=http
 # S3_HOSTNAME=192.168.1.123:9000
 
+# S3 (Minio Config (optional))
+# S3_ENABLED=true
+# S3_BUCKET=check minio's first launch for details
```

Comment:
I suggest moving "check minio's details etc" text into a comment line/block. The commented out variables should still contain valid (or empty) values.